scipy/stats/_qmc.py

```diff
@@ -950,18 +968,56 @@ class LatinHypercube(QMCEngine):
            [6.80338249, 3.08795949],
            [2.65448791, 3.83491828]])
 
+    Alternatively, optimized LHS can be constructed with `optimization`.
```

Comment:
This is not an alternative to scaling, so it should either go before scaling or the word "Alternatively" should be changed or dropped.